### PR TITLE
Update for 2017-10-12, adding Enterprise updates to the list.

### PR DESCRIPTION
**Pull request by DrTorte**
Flagging for review, but updates enterprise items and a few other little tidbits.

### CODE REVIEW

File: _data/steps.yml [hunk 1]
```diff
@@ -1,20 +1,48 @@
-
-- 5:
-  title: 'Improve Repository Directory Caching'
-  description: 'When running builds, directory caching should be fast, reliable and smart. We have deployed changes to use gzip vs bzip, and will be changing from mtime detection to md5 checksums.'
+- 17:
+  title: 'Enterprise: Trusty build containers'
+  description: 'Provide Ubuntu 14.04 (Trusty Tar) build images for Enterprise users.'
```
Comment (schultyy) [inline _data/steps.yml]: Ahh here it's `Trusty Tahr`. I believe I also mispelled it in my comment in that other ticket earlier. Sorry! 😞

**Review by schultyy — CHANGES_REQUESTED** [review]
Found one typo - except that LGTM

File: _data/steps.yml [hunk 2]
```diff
@@ -1,20 +1,48 @@
...
+- 19:
+  title: 'Enterprise: Configurable queues'
+  description: 'Users should be able to configure build queues to build specific projects only on certain machines'
+  icon: 'shipment'
+  status: 'Beta'
+  priority: 'normal'
+- 20:
+  title: 'Enterprise 2.2'
+  description: 'With Enterprise 2.2, we are adding many new features to our Enterprise services, including bringing some over from our hosted users'
+  status: 'In Progress'
```
Comment (acnagy) [inline _data/steps.yml]: we seem to be missing an icon here

**Review by acnagy — CHANGES_REQUESTED** [review]
Aside from the one missing icon, things are looking good for current stuff. I think, however, we should handle how we're dealing with the very old backlog - it's a little rough seeing info that we're doing customer chat... as that was stopped before I started. 

Perhaps we should either fill in the entire history between whatever was last updated and today _or_ go back to the beginning of this year and delete everything from before 2017?

**Pull request closed without merging**